script/preview.js

```diff
@@ -31,16 +34,28 @@ if (options.unexpected && options.unexpected.length !== 0) {
 }
 
 const app = express();
+const drupalClient = getDrupalClient(options);
 
 app.use(express.static(path.join(__dirname, '..', options.buildpath)));
 
-app.use('/preview', (req, res) => {
+app.use('/preview', async (req, res) => {
   const smith = createPipieline({
     ...options,
     port: process.env.PORT,
   });
 
   const contentId = req.query.contentId;
+  const pageData = await drupalClient.query({
```

Comment:
A `null` value for `route` but otherwise yeah -

![image](https://user-images.githubusercontent.com/1915775/50844485-d5593380-1338-11e9-8e68-6032d96d75a0.png)